spikeextractors/multirecordingchannelextractor.py

```diff
@@ -39,7 +39,9 @@ def __init__(self, recordings, groups=None):
                 for i, group in enumerate(groups):
                     recording = recordings[i]
                     channel_ids = recording.get_channel_ids()
-                    group_values += [group] * len(channel_ids)
+                    recording_groups = [group] * len(channel_ids)
+                    group_values += recording_groups
+                    recording.set_channel_groups(groups=recording_groups)
```

Comment:
@colehurwitz as per our discussion about the internal extractors, I think we can remove this
```suggestion
                   
```

Reply:
Agreed!